doc/nrf/ug_zigbee_commissioning.rst

```diff
@@ -0,0 +1,214 @@
+This happens by using install code link keys specific to the joining device.
+The install code link key is exchanged with the Trust Center, which allows for encrypting packets exchanged between the Trust Center and the new node.
+This way, only the Trust Center and the new node can decrypt the packages exchanged between them.
+
+Distributed security network
+----------------------------
+
+The distributed security network is formed by Routers and does not have a centralized node that manages security in the network.
+In this network type, all Routers are equal and there is no root of trust between them.
+Each Router can provide the network key to the joining device.
+
+.. WIP
+
+   * Related to Touchlink -- routers set up an encrypted channel with other devices and share data
+
+Finding and Binding
+===================
+
+After the node joins a network, it can undergo the Finding and Binding procedure.
+With this procedure, you can establish connection between corresponding endpoints on two different nodes, which improves the communication speed between these nodes.
+This is because, once bound to each other, the nodes do not have to use the device address to communicate, but can rely on connections between the same clusters in their binding tables.
+
+.. figure:: /images/zigbee_commissioning_fb.svg
+   :alt: Zigbee Finding and Binding procedure
+
+   Zigbee Finding and Binding procedure
+
+Finding and Binding can be started automatically (using the ``bdbCommissioningMode`` mask) or manually.
+In this mode, the joining node is the target device, while the node already in the network is the initiator device.
+The target node identifies itself and responds to queries from the initiator.
+The initiator starts Finding and Binding and periodically searches for endpoints on target.
+When it receives a response, it identifies the endpoint on target with a ZDO simple descriptor request and creates connections with the clusters corresponding to the ones on the initiator.
+This connection can either be a binding (entry in the binding table) or a grouping (addition of the target node address to the address group).
+The procedure is performed using ZDO simple descriptor requests and responses for each endpoint pair, for each matching Zigbee cluster client and server pair for the endpoint.
+
+
+Finding and Binding can be terminated if there are no corresponding clusters on the target node.
+
+For example, in the :ref:`Zigbee quick start guide <ug_zigee_qsg>` network made of one Coordinator, one Router (light bulb), and one End Device (light switch), when the Coordinator forms a network, the endpoints on the Router and the End Device create bindings with the endpoints on the Coordinator node.
+This allows you to control the light bulb using the light switch, because the compatible endpoints are bound.
+
+End of commissioning (node reset)
+*********************************
+
+* Reset can be achieved
```

Comment:
Small thing, but perhaps this can be changed to:

```
* Reset can be achieved via:

   * Identity cluster
   * Touchline
   * Local interaction